content/blog/2021/10/2021-10-31-hacktoberfest-results-2021.adoc

```diff
@@ -0,0 +1,88 @@
+We thank all the Hacktoberfest contributors and the maintainers who reviewed the submitted pull requests.
+
+We received contributions in artwork, translation, documentation, security, and general purpose improvements.
+The contributions included software improvements, documentation updates, and video tutorials.
+
+== Translations and Artwork
+
+link:https://www.duchess-france.org/[Duchess France] provided significant improvements to the French localization of Jenkins.
+The changes included new translations of text, corrections of existing translations, and file encoding fixes.
+The translation changes included link:https://github.com/jenkinsci/jenkins/pulls?q=is%3Apr+is%3Aclosed+created%3A%3E2021-10-01+french[fifteen pull requests] representing the work of six different contributors.
+
+Translation pull requests were also received for Spanish language improvements.
+The translation pull requests included multiple plugins in addition to Jenkins core.
+
+A contributor from link:https://www.duchess-france.org/["Duchess France [Women in tech\]"] created the first Jenkins logo highlighting a woman.
+Special thanks to link:https://github.com/tatoberres[tatoberres] for the new artwork!
+
+image:/images/logos/duchess/256.png[height=200]
+
+link:https://github.com/highflyer910[Thea Mushambadze] created the new Jenkins logo for Georgia logo.
+
+image:/images/logos/georgia/georgia.png[ height=200 ]
+
+Thanks very much!
+
+== Plugin Docs Migration to GitHub
+
+The Jenkins project started the migration of plugin documentation to GitHub almost two years ago.
+Hacktoberfest 2021 identified 40 candidate plugins to migrate their documentation to GitHub.
+link:https://github.com/Dan-Heath[Dan Heath],
+link:https://github.com/Mr-DG-Wick[Deepak Gupta],
+link:https://github.com/avinashupadhya99[Avinash Upadhyaya K R],
+link:https://github.com/rajanssingh[Rajan Kumar Singh], and
+link:https://github.com/dheerajodha[Dheeraj Singh Jodha] provided a total of 29 pull requests migrating plugin documentation to GitHub.
+
+Special thanks to the contributors and to the plugin maintainers that merged and released documentation migration pull requests.
+Eight plugins have fully migrated their documentation to GitHub through Hacktoberfest contributions.
+An additional thirteen plugins have merged the documentation change and will complete the migration with their next plugin release.
+
+The plugin documentation migration was coordinated through a link:https://docs.google.com/spreadsheets/d/1xxz6v-N6h17PI9Qu9uKrWaiKdI94StfeTfzWbwIYsX8/edit?usp=sharing[worksheet].
+A link:https://www.youtube.com/watch?v=NU2g373wHNo&t=2331s[tutorial video] describing the documentation migration process is also available.
+
+== Implementing Content Security Policy
+
+Wadeck Follonier provided a link:https://www.youtube.com/watch?v=NU2g373wHNo&t=3369s[tutorial] to prepare Jenkins for a broader implementation of Content Security Policy.
```

Comment:
This link is taking me to a .png file that shows a long, skinny image of this blog rather than the tutorial.  And is a video a tutorial?  I might prefer verbiage like "Wadeck Follonier provided the <link>[Title of Video] video that shows how to prepare Jenkins for a broader implementation of Content Security Policy that is upcoming."  I like links that include a title that I will see when I follow the link...

Reply:
That's really strange.  When I open https://www.youtube.com/watch?v=NU2g373wHNo&t=3369s , it positions me at the start of Wadeck's instructions on Content Security Policy and moving JavaScript from inside jelly files to separate files.

Could you check again?

Reply:
Meg was clicking the link from the preview image.  The link is correct.